public/manifest.json

```
@@ -0,0 +1,21 @@
+{
+  "name": "ClimbUp",
+  "short_name": "ClimbUp",
+  "description": "ClimbUp es una plataforma donde se reunen y gestionan las competencias de escalada",
```

```suggestion
  "description": "ClimbUp es una plataforma especializada en la gestión de competencias de escalada, diseñada para optimizar la organización de eventos",
```

Eso es lo que tenemos puesto en el Notion